### DISCUSSION TITLE
Option to transfer OLS/RLS/Perspectives when a new table is imported

### DISCUSSION THREAD

**Discussion by lukiz84** [TabularEditor/TabularEditor3#1191] [Ideas]
### Is your feature request related to a problem? Please describe

When I add a new table which should have the same OLS/RLS/Perspective settings as an already imported table I have to set all the settings manually (which is ok, because how would TE3 know what I want)

### Describe the solution you'd like

I'd like a context menu entry for the existing table with something like "Copy metadata" or so and then be able to paste it to the new table

### Goals

1. Quicker development

**Comment by mlonsk** [comment]
Hi @lukiz84 
I like the idea of having specialized paste functions and can see how that can be very helpful!

For now though I would recommend the following workaround:
1. Copy/Paste an existing table
2. Update partition to point to the new table
3. Make a Schema Update by right clicking on the new table and choosing Schema Update
You need to ensure that the clipboard preferences are set to copy what you require: 
![image](https://github.com/TabularEditor/TabularEditor3/assets/11939114/df4a2ead-074d-4fc1-8f1a-a40fa8df964f)

Would that work for your use case?
